### PR TITLE
Add mandatory version param for rest#request.

### PR DESCRIPTION
**Pull request by SimonWoolf**
Adds mandatory version param to rest#request for protocol version 2.0, per https://ably.atlassian.net/wiki/spaces/ENG/pages/2222456951/ADR61+Versioning+of+realtime+API+endpoints

### CODE REVIEW

File: textile/features.textile
Comment (QuintinWillison) [inline textile/features.textile:132]: If removing this spec point then we should replace it with a message similar to the one you've replaced `RSC19a` with.

File: textile/features.textile
Comment (QuintinWillison) [inline textile/features.textile:132]: I realise that some of this is inherited from `RSC19a`, however this is a new spec point so perhaps worth fixing now also... there's a mix of `string` versus `String` in the method signature. I think we should conform to one or the other, presumably `String` as that's what's in the IDL.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]

File: textile/features.textile
Comment (SimonWoolf) [reply textile/features.textile:132]: Did not intend to remove that spec item (and the [old docs pr](https://github.com/ably/docs/pull/1592/files) doesn't, so possibly a glitch when porting to the other repo), I'll fix.

File: textile/features.textile
Comment (SimonWoolf) [reply textile/features.textile]: done

**Review by QuintinWillison — APPROVED** [review]
Looks good to me, but I'll add a couple of other reviewers so we can perhaps get more than just my approval before we land this.

**Review by owenpearson — APPROVED** [review]

**Review by stmoreau — APPROVED** [review]

**Pull request merged**
